archivers/zlib/Portfile

```diff
@@ -48,7 +50,7 @@ test.run                yes
 destroot.destdir        prefix=${destroot}${prefix}
 
 if {${name} eq ${subport}} {
-    revision            0
+    revision            2
```

Comment:
It looks like you're bumping the revision by 2, both here, as well as for `minizip`.

This should be 1, and the other 2.

Reply:
@mascguy given the changes in previous commits, won't this cause conflicts?

Reply:
> @mascguy given the changes in previous commits, won't this cause conflicts?

Which commit(s) are you referring to...?

Reply:
NVMD, just a little confused.